tests/network/localnet/migration_stuntime/libstuntime.py

```
@@ -0,0 +1,142 @@
+import re
+from typing import Final, Self
+
+from libs.vm.vm import BaseVirtualMachine
+from tests.network.libs.connectivity import build_ping_command
+
+LOGGER = logging.getLogger(__name__)
+
+SERVER_VM_LABEL: Final[tuple[str, str]] = ("stuntime.test", "server")
+CLIENT_VM_LABEL: Final[tuple[str, str]] = ("stuntime.test", "client")
+STUNTIME_THRESHOLD_SECONDS: Final[float] = 5.0
+STUNTIME_PING_LOG_PATH: Final[str] = "/tmp/stuntime-ping.log"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/stuntime-ping.log"

(S108)